src/ast/callback_visitor.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include "ast.h"
+#include <functional>
+
+namespace bpftrace {
+namespace ast {
+
+using callback = std::function<void(Node *, void *)>;
+
+class CallbackVisitor : public Visitor
+{
+public:
+  explicit CallbackVisitor(callback func, void *data) : func_(func), data_(data)
```

Comment:
I think `explicit` is only necessary if you have a single arg constructor and you want to prevent implicit conversions.

Reply:
ok, remove it.